packages/auth/amplify_auth_cognito_test/test/flows/srp/srp_password_verifier_worker_test.dart

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO(dnys1): Investigate DDC failures
+@Tags(['no-ddc'])
```

Comment:
nit: Would we ever consider keeping tags in a centralized place as consts/enums? Like maybe in amplify_test? That way they're easier to keep track of/not repeat, etc...

Reply:
This is definitely temporary. At some point, if we have more permanent tags used throughout, that could definitely make sense.